content/de/docs/intro/using_helm.md

```diff
@@ -96,6 +96,8 @@ Am einfachsten hat das Kommando zwei Argumente: Einen Versionsnamen, den Sie gew
 haben und den Namen des Charts, dass Sie installieren möchten.
 
 ```console
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm repo update
```

Comment:
The call to `helm repo update` after a `helm repo add` is redundant. Would you mind removing this?